src/app/ui-library/input.tsx

```diff
@@ -0,0 +1,236 @@
+
+type InputVariant = 'danger' | 'default';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const TitleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: ${(props) => props.theme.space.xs};
+`;
+
+const Title = styled.div`
+  ${(props) => props.theme.typography.body_medium_m}
+  color: ${(props) => props.theme.colors.white_200};
+`;
+
+const InfoText = styled.div`
+  ${(props) => props.theme.typography.body_medium_m}
+  color: ${(props) => props.theme.colors.white_0};
+`;
+
+const InputContainer = styled.div`
+  position: relative;
+`;
+
+const InputField = styled.input<{ $variant: InputVariant }>`
+  ${(props) => props.theme.typography.body_medium_m}
+
+  width: 100%;
+
+  color: ${(props) => props.theme.colors.white_200};
+  background-color: ${(props) => props.theme.colors.elevation_n1};
+  padding: ${(props) => props.theme.spacing(5)}px ${(props) => props.theme.spacing(8)}px;
+  border-radius: 8px;
+
+  caret-color: ${(props) => props.theme.colors.tangerine};
+
+  ${(props) => {
+    switch (props.$variant) {
+      case 'danger':
+        return `
+          border: 1px solid ${props.theme.colors.danger_dark_200};
+        `;
+      default:
+        return `
+          border: 1px solid ${props.theme.colors.white_800};
+        `;
+    }
+  }}
+
+  &:focus {
+    ${(props) => {
+      switch (props.$variant) {
+        case 'danger':
+          return `
+            border: 1px solid ${props.theme.colors.danger_dark_200};
+          `;
+        default:
+          return `
+            border: 1px solid ${props.theme.colors.white_600};
+          `;
+      }
+    }}
+  }
+
+  ::placeholder {
+    color: ${(props) => props.theme.colors.white_400};
+  }
+`;
+
+const ComplicationsContainer = styled.div`
+  position: absolute;
+  right: ${(props) => props.theme.spacing(4)}px;
+  top: 0;
+
+  height: 100%;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const ClearButton = styled.div<{ $hasSiblings?: boolean }>`
+  cursor: pointer;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+
+  width: 16px;
+  height: 16px;
+  background-color: ${(props) => props.theme.colors.white_200};
+  color: ${(props) => props.theme.colors.elevation_n1};
+  border-radius: 50%;
+  margin-right: ${(props) => (props.$hasSiblings ? props.theme.spacing(4) : 0)}px;
+
+  :before,
+  :after {
+    position: absolute;
+    left: 7px;
+    content: ' ';
+    height: 8px;
+    width: 2px;
+    background-color: black;
+  }
+  :before {
+    transform: rotate(45deg);
+  }
+  :after {
+    transform: rotate(-45deg);
+  }
+`;
+
+const SubText = styled.div`
+  color: ${(props) => props.theme.colors.white_400};
+  margin-top: ${(props) => props.theme.spacing(4)}px;
+
+  ${(props) => props.theme.typography.body_s}
+`;
+
+const Feedback = styled.div``;
```

Comment:
Hey, I think we need a little margin here.

```suggestion
const Feedback = styled.div`
  margin-top: 8px;
`;
```

<img width="393" alt="image" src="https://github.com/secretkeylabs/xverse-web-extension/assets/65149726/dec89a85-a376-4149-8d4e-d8f5d95f548b">
<img width="384" alt="image" src="https://github.com/secretkeylabs/xverse-web-extension/assets/65149726/094442e4-8833-407e-b8c5-60fa051b6b5c">
<img width="960" alt="image" src="https://github.com/secretkeylabs/xverse-web-extension/assets/65149726/1e8d2ea5-c0cd-4707-9870-b6d655981c4c">

Reply:
Oops, thanks :)